tests/test_server.py

```diff
@@ -593,6 +593,58 @@ async def test_kp_not_trapi():
     assert 'Received non-TRAPI compliant response from KP' in output['logs'][0]['message']
 
 
+@pytest.mark.asyncio
+@with_norm_overlay(settings.normalizer_url)
+@with_registry_overlay(
+    settings.kpregistry_url, {
+        'ctd': {
+            'url':
+            'http://ctd/query',
+            'operations': [{
+                'source_type': 'biolink:ChemicalSubstance',
+                'edge_type': '-biolink:treats->',
+                'target_type': 'biolink:Disease'
+            }],
+            'details': {
+                'preferred_prefixes': {}
+            }
+        }
+    })
+@with_response_overlay(
+    "http://ctd/query",
+    Response(
+        status_code=200,
+        content=json.dumps({"message": {
+            "query_graph": None,
+            "knowledge_graph": None,
+            "results": None,
+        }}),
+    )
+)
+async def test_kp_response_empty():
```

Comment:
The naming and documentation of this test don't match with what you're asserting. Are no `logs` indicative of an empty response with null query graph?

Reply:
It's tricky because Strider is pretty resilient w.r.t. runtime errors. Prior to this fix, an error was encountered in our code when the KP response was missing the query graph, then the error was caught and was logged. This fixes the error and thus prevents the logged message.